casaos-fix-docker-api-version/test-script.sh

```
@@ -420,10 +425,14 @@ test_fix_script() {
   local after_version=$(get_docker_version)
   local after_api=$(get_docker_api_version)
   local dockerd_binary=$(get_dockerd_binary_version)
+  local containerd_pkg=$(dpkg -l 2>/dev/null | grep containerd.io | awk 'NR==1 {print $3}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)